src/sources/kafka.rs

```diff
@@ -56,6 +56,14 @@ enum BuildError {
     KafkaSubscribeError { source: rdkafka::error::KafkaError },
 }
 
+/// Metrics configuration.
+#[configurable_component]
+#[derive(Clone, Debug, Default)]
+struct Metrics {
+    /// Expose topic lag metrics (`kafka_consumer_lag`).
```

Comment:
Can we also say that it's for every topic and partition?  This description still seems a bit opaque.

Reply:
Rephrased a little bit :)